app/src/main/java/Bamboo/view/game/Canvas.java

```diff
@@ -14,13 +16,15 @@
 import java.awt.Dimension;
 import java.awt.Graphics;
 import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import java.util.HashMap;
 
 public class Canvas extends JPanel
 {
     private Game game;
 
     private static int circle_radius = 50;
-    private BasicStroke circle_thickness = new BasicStroke(2);
+    //private BasicStroke circle_thickness = new BasicStroke(2);
```

Comment:
This has been commented out, but why? 
- Is it no longer used? -> then delete it
- Is it being kept for a reason -> say why in a comment and who is responsible (Use `//TODO` to assign this to somebody)